### PR TITLE
[Fix] Link preview not working for Youtube links 

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?
https://wearezeta.atlassian.net/browse/SQSERVICES-67

### Issues
User doesn't see a link previews for Youtube links. 

### Causes
There is a limit on the amount of data we want to fetch. 
Some websites in the wild don’t have the closing `</head> tag`, so this was a way to reduce the amount of data that needs to be downloaded. We check if there is a closing ` </head> tag `or the download data is less than the set limit.
As a result, we was canceling the task of fetching data without getting all necessary data (all metadata)

### Solutions
Solution is to remove the current fetch limit. This was done because we assume that there will be more cases when the amount of data is bigger than we expected (and it's not easy to predict the average amount of data) than websites without closing `</head> tag`.

### CODE REVIEW

File: WireLinkPreviewTests/IntegrationTests.swift [hunk 1]
```diff
@@ -64,7 +64,7 @@ class IntegrationTests: XCTestCase {
     }
     
     func testThatItParsesSampleDataVimeo() {
-        let expectation = OpenGraphDataExpectation(numberOfImages: 1, type: "video", siteNameString: "Vimeo", userGeneratedImage: false, hasDescription: true, hasFoursquareMetaData: false)
+        let expectation = OpenGraphDataExpectation(numberOfImages: 1, type: "video.other", siteNameString: "Vimeo", userGeneratedImage: false, hasDescription: true, hasFoursquareMetaData: false)
```
Comment (KaterinaWire) [inline WireLinkPreviewTests/IntegrationTests.swift:67]: These changes are not related to this PR. I just saw a failed test.

**Review by typfel — APPROVED** [review]

**Review by David-Henner — APPROVED** [review]

**Comment by KaterinaWire** [comment]
@zenkins release patch
